examples/Bezier-curves.qmd

```diff
@@ -174,7 +174,7 @@ pB_opt = gradient_descent(
         contraction_factor=0.5,
         sufficient_decrease=0.001,
     ),
-    stopping_criterion=StopWhenChangeLess(1e-5) |
+    stopping_criterion=StopWhenChangeLess(N, 1e-5) |
```

Comment:
I think with the factory, this is not 100% necessary? IO hope it is not, because that was the main Goan of the factory ;)

Reply:
I'm not sure what's happening here. The failure seems to be related to the storage in `StopWhenChangeLess`. But `StopWhenChangeLess` doesn't seem to be converted to the factory approach anyway.

Reply:
Will take a look the next few days then, because it should I think.

Reply:
it seems necessary indeed.